### PR TITLE
Feature/INTG-729

### PR DESCRIPTION
**Pull request by jwsandeman**

### CODE REVIEW

File: src/components/CustomContentPanel/CustomContentPanel.tsx [hunk 1]
```diff
@@ -2,13 +2,27 @@ import React, { useState } from 'react';
...
 import { ReactComponent as TickIcon } from '../../images/TickIcon.svg';
 import { ReactComponent as CopyIcon } from '../../images/CopyIcon.svg';
 
-export default function ReceiptPanel({ children, title, css, textReceipt }: IReceiptPanel): React.ReactElement {
-  const classes = useReceiptPanelStyles();
+export type CustomContentPanelProps = {
+  children: React.ReactNode;
+  title: string;
+  css: string;
+  controlledProps?: {
```
Comment (rossi-mx51) [inline src/components/CustomContentPanel/CustomContentPanel.tsx:13]: You can simply put content and copiable as a normal props. We only use this controlledProps pattern if we have something similar in the local state, but we want it to be control by the users

File: src/components/CustomTabPage/CustomTabPage.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  customGridPanel,
+  ...other
+}: CustomTabPageProps) => {
+  const showFlowPanel = useAppSelector(selectedShowFlowPanel);
+  const classes = useStyles({ showFlowPanel });
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
```
Comment (rossi-mx51) [inline src/components/CustomTabPage/CustomTabPage.tsx:31]: does this have any effect ?

File: src/components/PurchaseFlow/PurchaseFlow.tsx [hunk 3]
```diff
@@ -13,6 +13,7 @@ import currencyFormat from '../../utils/common/intl/currencyFormatter';
 export default function PurchaseFlow(): React.ReactElement {
   const selectedTerminal = useSelector(selectedTerminalIdSelector);
   const currentTerminal = useSelector(terminalInstance(selectedTerminal)) as ITerminalProps;
+  console.log('currentTerminal test=>', currentTerminal);
```
Comment (rossi-mx51) [inline src/components/PurchaseFlow/PurchaseFlow.tsx:16]: forgot to remove ?

File: src/components/SupportToolsPage/AutoAddressCheck/index.tsx [hunk 4]
```diff
@@ -272,12 +272,13 @@ function AutoAddressCheck(): React.ReactElement {
   const onEnvironmentChange = (e: IFormEventValue) => setSelectedEnvironment(e.target.value as string);
 
   return (
-    <Box className={classes.toolContainer}>
-      <Typography component="h1" className={classes.h1}>
-        Auto address check
-      </Typography>
-      <Typography>This support tool can be used by Merchants or L2 support users to test Auto address</Typography>
-      <Divider className={classes.divider} />
+    // <Box className={classes.toolContainer}>
```
Comment (rossi-mx51) [inline src/components/SupportToolsPage/AutoAddressCheck/index.tsx:275]: If this is not used anymore, why not delete it ?

File: src/components/CustomTabPage/CustomTabPage.tsx [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+interface CustomTabPageProps {
+  children?: React.ReactNode;
+  index: number;
+  value: number;
+  title: string;
+  subtitle: string;
+  customGridPanel?: React.ReactNode;
+}
+
+export const CustomTabPage = ({
```
Comment (rossi-mx51) [inline src/components/CustomTabPage/CustomTabPage.tsx:16]: Is this a page or a component ?

File: src/components/SupportToolsPage/AutoAddressCheck/AutoAddressCheckPage.tsx [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { CustomTabPage } from '../../CustomTabPage/CustomTabPage';
+import AutoAddressCheck from '.';
+
+export const AutoAddressCheckPage: React.FC<{ tabIndex: number }> = ({ tabIndex }) => (
```
Comment (rossi-mx51) [inline src/components/SupportToolsPage/AutoAddressCheck/AutoAddressCheckPage.tsx:5]: Same as above, is this a page or a component ?

File: src/services/spiService/index.ts [hunk 7]
```diff
@@ -914,6 +914,11 @@ class SpiService {
     const spi = this.setSpiConfig(instanceId);
     return spi.AcceptSignature(false);
   }
+
+  initiateGetTransaction(instanceId: string, posRefId: string) {
+    const spi = this.setSpiConfig(instanceId);
```
Comment (rossi-mx51) [inline src/services/spiService/index.ts:919]: Why do you need this ?

**Review by rossi-mx51 — CHANGES_REQUESTED** [review]

File: src/components/CustomTabPage/CustomTabPage.tsx (hunk 5 above)
Comment (jwsandeman) [reply src/components/CustomTabPage/CustomTabPage.tsx]: Changed to CustomTab.

File: src/components/PurchaseFlow/PurchaseFlow.tsx (hunk 3 above)
Comment (jwsandeman) [reply src/components/PurchaseFlow/PurchaseFlow.tsx]: Good pickup, removed.

File: src/components/SupportToolsPage/AutoAddressCheck/index.tsx (hunk 4 above)
Comment (jwsandeman) [reply src/components/SupportToolsPage/AutoAddressCheck/index.tsx]: Good pickup, removed.

File: src/components/CustomContentPanel/CustomContentPanel.tsx (hunk 1 above)
Comment (jwsandeman) [reply src/components/CustomContentPanel/CustomContentPanel.tsx]: Ok no worries, removed controlledProps.

File: src/components/CustomTabPage/CustomTabPage.tsx (hunk 2 above)
Comment (jwsandeman) [reply src/components/CustomTabPage/CustomTabPage.tsx]: Don't think so, removed.

**Review by jwsandeman — COMMENTED** [review]
Ok ive made those changes now.

**Review by rossi-mx51 — APPROVED** [review]

**Pull request merged**